src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -939,6 +939,34 @@ async def WriteAttribute(self, nodeid: int, attributes: typing.List[typing.Tuple
             future, eventLoop, device.deviceProxy, attrs, timedRequestTimeoutMs=timedRequestTimeoutMs, interactionTimeoutMs=interactionTimeoutMs, busyWaitMs=busyWaitMs).raise_on_error()
         return await future
 
+    def WriteGroupAttribute(self, groupid: int, attributes: typing.List[typing.Tuple[ClusterObjects.ClusterAttributeDescriptor, int]], busyWaitMs: typing.Union[None, int] = None):
+        '''
+        Write a list of attributes on a target group.
+
+        groupid: Group ID to send write attribute to.
+        attributes: A list of tuples of type (cluster-object, data-version):
```

Comment:
data-version is allowed to be omitted, right?

Reply:
Correct, updated comment to reflect that it can be omitted